### PR TITLE
Commerce 4669 category changes

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -296,11 +296,13 @@ category-display-page-help=The display page is used to display the category.
 category-display-pages=Category Display Pages
 category-selected=Category Selected
 category.catalog=Catalog
+category.data-integration=Data Integration
 category.google-merchant=Google Merchant
 category.inventory=Inventory
 category.orders=Orders
 category.payment=Payment
 category.pricing=Pricing
+category.salesforce-talend-job-deployer=Salesforce Connector
```
Comment (marco-leo) [inline modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties:305]: @dsitu Is correct to have label diffeent than the text?

Comment (dsitu) [reply modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties:305]: @marco-leo andrea-ale-sbarra hmm, I think we should try to keep them the same if we can.
So `category.salesforce-talend-job-deployer=Salesforce Talend Job Deployer` or `category.salesforce-connector=Salesforce Connector`

Comment (marco-leo) [reply modules/apps/commerce/commerce-lang/src/main/resources/content/Language.properties:305]: let's go with category.salesforce-connector=Salesforce Connector

**Review by marco-leo — CHANGES_REQUESTED** [review]
Please change labels

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**